express.js

```diff
@@ -1,63 +1,108 @@
 const express = require('express');
 const app = express();
+require('dotenv').config();
+const { v4: uuidv4 } = require('uuid');
 const { readUsers, writeUsers } = require('./helpers.js');
-
 app.use(express.json());
-const PORT = 3054;
+app.set('PORT', process.env.PORT || 3080);
+
+   // TODO add id as first level obecht key in json
 
-app.get('/users', (req, res) => {
-    let info = readUsers();
-    res.json(info.users);
+app.get('/users/:id', (req, res) => {
+    let users = readUsers();
+    let param = req.params.id;
```

Comment:
use destructuring always
```suggestion
    let { id } = req.params;
```